pliant/README.md

```diff
@@ -117,7 +117,7 @@ Need help? Contact [Datadog support][7].
 
 [1]: https://pliant.io/
 [2]: https://docs.datadoghq.com/agent/kubernetes/integrations/
-[3]: https://github.com/DataDog/integrations-core/blob/master/pliant/datadog_checks/pliant/data/conf.yaml.example
+[3]: https://github.com/DataDog/integrations-extras/blob/master/pliant/datadog_checks/pliant/data/conf.yaml.example
```

Comment:
This link 404's. I don't see any file like this in the pliant directory.

Reply:
😓 sorry about that, I did not check if the link was actually used :/ Fixed.